### PR TITLE
Cockpit 283

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-283.html

### CODE REVIEW

**Comment by garrett** [comment]
GitHub keeps not building the site, oddly... so the preview link will be broken until it does.

**Comment by garrett** [comment]
OK, I tried a few times and finally got GitHub to update the preview link.

File: _posts/2023-01-11-cockpit-283.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 283 and cockpit-machines 281:
+
+
+## Services: Create timer to run every minute
```
Comment (KKoukiou) [inline _posts/2023-01-11-cockpit-283.md:17]: let's add special thanks to the upstream contributor for that. It's https://github.com/dphaas2004

Comment (garrett) [reply _posts/2023-01-11-cockpit-283.md:17]: Great suggestion! It's always good to have shout-outs to contributors.

File: _posts/2023-01-11-cockpit-283.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: Cockpit 283
+author: garrett
+date: '2023-01-11'
+tags: cockpit, machines
+slug: cockpit-283
+category: release
+summary: ''
+---
+
```
[the remark this answers is not in the document]
Comment (garrett) [reply _posts/2023-01-11-cockpit-283.md:10]: No, I didn't... the preview mode has merged items with the release-notes tag. The non-preview mode only includes merged items with the release-notes tag. When we're doing a release, we don't include PRs that aren't merged yet. (The script handles this.)

**Comment by garrett** [comment]
Re-ran the release notes script, ran oxipng on the images, and force-pushed the results.

File: _posts/2023-01-12-cockpit-283.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Cockpit Machines now has support for adding a virtual software-based watchdog device to a machine.
+
+Please note that the watchdog device _also_ needs to be configured within the guest operating system to be used.
+
+![screenshot of virtual watchdog device support]({{ site.baseurl }}/images/283-machines-virtual-watchdog-device-support.png)
+
+---
+
+Things to do after this is merged:
```
Comment (KKoukiou) [inline _posts/2023-01-12-cockpit-283.md:41]: This needs to be removed

File: _posts/2023-01-12-cockpit-283.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+## Try it out
+
+Cockpit 283 and cockpit-machines 281 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/283)
```
Comment (KKoukiou) [inline _posts/2023-01-12-cockpit-283.md:56]: Can you please fix the links. They all exist now.

**Review by KKoukiou — COMMENTED** [review]
One comment and links fix and it's good from my side.

File: _posts/2023-01-12-cockpit-283.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+Things to do after this is merged:
+
+ * [ ] Create a libvirt issue requiring all watchdog actions to have relevant VM events (it seems that currently only watchdog reset and watchdog force-off have events)
+ * [ ] Create a libvirt issue asking for API to check VM's history of watchdog events (this would allow us to see if VM is stuck in a reboot loop)
+ * [ ] Create an issue to implement a warning if VM is crashed
+ * [ ] Redesign VM's Overview tab since it's getting crowded
+
+
```
Comment (garrett) [inline _posts/2023-01-12-cockpit-283.md:48]:
```suggestion
```

File: _posts/2023-01-12-cockpit-283.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/283)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/281)
+* [cockpit-machines Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit-machines)
+* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit-machines)
```
Comment (garrett) [inline _posts/2023-01-12-cockpit-283.md:61]:
```suggestion
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-74586bb92c)
* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-8bc12f90dd)
* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/281)
* [cockpit-machines Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-49b3964047)
* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-3b424a9b80)
```

File: _posts/2023-01-12-cockpit-283.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 283 and cockpit-machines 281:
+
+
+## Services: Create timer to run every minute
+
+In the Timers part of Services page, the create timer dialog can create systemd timers to run in "minutely" mode, which runs a command every minute on the specified second.
```
Comment (garrett) [inline _posts/2023-01-12-cockpit-283.md:19]:
```suggestion
In the Timers part of Services page, the create timer dialog can create systemd timers to run in "minutely" mode, which runs a command every minute on the specified second.

_Huge thanks to [Daniel Haas (dphaas2004)](https://github.com/dphaas2004) for [this feature](https://github.com/cockpit-project/cockpit/pull/17592#issue-1317035572)!_
```

**Comment by garrett** [comment]
I think it's ready for a final review now! :partying_face:

**Review by KKoukiou — APPROVED** [review]

**Review by KKoukiou — COMMENTED** [review]
@garrett can we also include the podman 60 in the urls and title? 
No notes needed.

**Comment by garrett** [comment]
I've manually added cockpit-podman 60 to the release notes and did a force push.

**Review by KKoukiou — APPROVED** [review]

**Review by martinpitt — APPROVED** [review]
LGTM, thanks to both of you!

**Pull request merged**